engine/footer.php

```diff
@@ -1,11 +1,9 @@
 <footer>
 	&copy; Znote AAC.
 	<?php
-	$time = microtime();
-	$time = explode(' ', $time);
-	$time = $time[1] + $time[0];
+	$finish = microtime(true);
 	$finish = $time;
```

Comment:
Double declaration of `$finish`?
Considering you just removed `$time`, this looks like a bug.

Reply:
Right, I forgot to remove that line. So why it was merged? :D